PocketKit/Sources/PocketKit/MyList/ItemsList/SwiftUI/Components/OverflowMenu.swift

```diff
@@ -2,37 +2,97 @@ import SwiftUI
 import Textile
 
 struct OverflowMenu: View {
-    var overflowActions: [ItemAction]
-    var trackOverflow: ItemAction?
-    var image: ImageAsset = .overflow
-    var trailingPadding: Bool = false
+    @EnvironmentObject
```

Comment:
In the future, lets see how we can do this in a way that enables SwiftPreviews.

Reply:
yes, should I create a ticket for this or keep in mind when creating SwiftUI views in the future?